test/e2e/tests/request-queuing/dapp1-switch-dapp2-send.spec.js

```diff
@@ -121,31 +112,39 @@ describe('Request Queuing Dapp 1, Switch Tx -> Dapp 2 Send Tx', function () {
           `window.ethereum.request(${switchEthereumChainRequest})`,
         );
 
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
+        await driver.findElement({
+          text: 'Allow this site to switch the network?',
+          tag: 'h3',
+        });
+
         await driver.switchToWindowWithUrl(DAPP_ONE_URL);
 
         await driver.clickElement('#sendButton');
 
-        await switchToNotificationWindow(driver, 4);
-        await driver.findClickableElements({
-          text: 'Switch network',
-          tag: 'button',
-        });
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
 
         await driver.clickElement({ text: 'Switch network', tag: 'button' });
 
+        await driver.switchToWindowWithUrl(DAPP_ONE_URL);
+
         // Wait for switch confirmation to close then tx confirmation to show.
-        await driver.waitUntilXWindowHandles(3);
```

Comment:
this point is where test breaks in webpack: if the next dialog window is opened really fast, this condition is never met